scaleway/resource_apple_silicon_server.go

```diff
@@ -48,7 +50,24 @@ func resourceScalewayAppleSiliconServer() *schema.Resource {
 				Description: "VNC url use to connect remotely to the desktop GUI",
 				Computed:    true,
 			},
-
+			"state": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Default:     applesilicon.ServerStatusStarting.String(),
+				Description: "The state of the server",
+				ValidateFunc: validation.StringInSlice([]string{
```

Comment:
I don't think it makes sense to validate that a user can put a M1 in an error state.

Reply:
The attribute will be read-only. we should keep the error in that case.

Reply:
No. If it is read only terraform does not have any value here. Let's suppose that apple silicon add a new state tomorrow, it will break with this code. Better to remove it all together.